examples/tests/invalid/function_type.test.w

```diff
@@ -12,3 +12,6 @@ interface IFace {
   inflight my_method2(x: num);
 //         ^^^^^^^^^^^^^^^^^^ Expected method return type
 }
+
```

Comment:
Could you add please a positive tests for the variadic case?

Reply:
yes, will do

Reply:
added